.php_cs

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->in(array(__DIR__))
+;
+
+return Symfony\CS\Config\Config::create()
+    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
+    ->fixers(array(
+        '-unalign_double_arrow',
```

Comment:
Yes I have the link: https://github.com/symfony/symfony/issues/12284  I remember the discussion well because I strongly opposed to this change.